src/GovUk.Education.ExploreEducationStatistics.Notifier/Utils/NotifierUtils.cs

```diff
@@ -0,0 +1,50 @@
+#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Notifier.Services;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.WindowsAzure.Storage.Table;
+using Notify.Client;
+
+namespace GovUk.Education.ExploreEducationStatistics.Notifier.Utils
+{
+    public static class NotifierUtils
+    {
+        private const string StorageConnectionName = "TableStorageConnString";
+        private const string NotifyApiKeyName = "NotifyApiKey";
+
+        public const string SubscriptionsTblName = "Subscriptions";
+        public const string BaseUrlName = "BaseUrl";
+        public const string WebApplicationBaseUrlName = "WebApplicationBaseUrl";
+        public const string TokenSecretKeyName = "TokenSecretKey";
+        public const string PendingSubscriptionsTblName = "PendingSubscriptions";
+
+        public const string ConfirmationEmailTemplateIdName = "SubscriptionConfirmationEmailTemplateId";
+        public const string VerificationEmailTemplateIdName = "VerificationEmailTemplateId";
+        public const string OriginalReleasePublicationEmailTemplateIdName =
+            "OriginalReleasePublicationEmailTemplateId";
+        public const string AmendedReleasePublicationEmailTemplateIdName =
+            "AmendedReleasePublicationEmailTemplateId";
```

Comment:
Would suggest moving these into a separate util class called `ConfigKeys` (or similar) to make it clearer what these are actually referring to.